modules/header/react/spec/__snapshots__/GlobalHeader.snapshot.tsx.snap

```diff
@@ -576,7 +576,7 @@ exports[`App GlobalHeader Snapshots renders a header with SystemIcon and IconBut
 `;
 
 exports[`App GlobalHeader Snapshots renders a header with a search bar 1`] = `
-.emotion-18 {
+.emotion-21 {
```

Comment:
Often these classname updates are because you didn't run `yarn build` before updating the tests. Can you try that first and see if it resolves this?

Reply:
running `yarn build` then `npm test -u` incremented the number again

Reply:
Yeah, I'm seeing the same thing. Trying to figure out what's going on. I'd prefer to figure it out before merging. I tried updating the snapshots with `--no-cache` as well, but no luck. Still investigating... 🔍